ng/src/web/components/dialog/savedialog.js

```diff
@@ -116,6 +126,7 @@ class SaveDialogContent extends React.Component {
             </ScrollableContent>
             <DialogFooter
               title={_('Save')}
+              loading={this.state.loading}
               onClick={() => this.handleSaveClick(state)}
```

Comment:
Could you change the onClick handler to
```jsx
onclick={() => !loading && this.handleSaveClick(state)}
```
?

Then the handler will not be called when loading is set to true.

Reply:
As an alternative you could add a check for loading to handleSaveClick e.g.
```javascript
if (onSave && !this.state.loading) {
```

Reply:
I went for the alternative solution.